jwst/tweakreg/tweakreg_step.py

```diff
@@ -132,7 +132,9 @@ def process(self, input):
                     for im in images:
                         filename = im.meta.filename
                         if filename in catdict:
-                            print(f"setting {filename}.tweakreg_catalog = {repr(catdict[filename])}")
+                            self.log.info(
+                                f"setting {filename}' tweakreg_catalog = {repr(catdict[filename])}"
```

Comment:
Is the formatting correct here? I see the entire string being enclosed in double quotes ("), but I also see a single quote (') right after {filename} that doesn't seem to have a partner to close it.